modules/ROOT/pages/consuming-a-soap-api.adoc

```diff
@@ -0,0 +1,20 @@
+= Consuming a SOAP API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint, studio, connectors, soap, wsdl, api
+
+If you want to consume an API that does not have its own Anypoint Connector, you can use the Web Service Consumer connector.
+
+Using the information contained in the API's WSDL, this connector enables you to configure a few details to establish the connection you need for consuming a service from within your Mule application:
+
+. Identify the location of the Web service's WSDL file.
+
+. Request the Web Service Consumer to configure itself from the WSDL – host, port, address.
```

Comment:
To be honest, the WSC does not configure itself. At design time it helps you to configure all those parameters giving you a list of options using the metadata retrieved from the WSDL, but the user has to configure all the parameters.
Besides, depending on the Web Service the user is connecting to, it may be necessary to configure other parameters related to security.
I'm saying this because there is a specification called WS-Policy where web services use XML to advertise their policies like security and allow clients to configure itself, but we don't support it yet.

Reply:
Updated.